docs/example-external.mdx

```diff
@@ -8,6 +8,8 @@ sidebar_label: External Examples
 
 - You can find more React Testing Library examples at
   [react-testing-examples.com](https://react-testing-examples.com/jest-rtl/).
+  
+  If the link does not work, try the netlify one: [react-testing-examples.netlify.app](https://react-testing-examples.netlify.app/).
```

Comment:
The original link may just be dead temporarily. Did you check with the project if they're aware of any issue?

The problem with the netlify link is that it's now tied to their deploy service. When they switch (e.g. to Vercel) the link is dead again. 

I'd rather ask the maintainers of https://react-testing-examples.com/ first for clarificiation. For example, our domain was dead a few days ago as well which was just a temporary netlify issue.